crates/sui-indexer-alt/src/args.rs

```diff
@@ -18,7 +18,16 @@ pub struct Args {
 #[derive(Subcommand, Clone, Debug)]
 pub enum Command {
     /// Run the indexer.
-    Indexer(IndexerConfig),
+    Indexer {
+        #[command(flatten)]
+        indexer: IndexerConfig,
+
+        /// Number of checkpoints to delay indexing summary tables for.
+        #[clap(long)]
+        consistent_range: Option<u64>,
```

Comment:
Why isn't this part of IndexerConfig?

Reply:
I was trying to separate configuration for our specific indexer from configuration for an indexer framework -- consistent range is something specific to our indexer.